src/string_utils.cpp

```diff
@@ -11,8 +11,9 @@
 
 bool lcmatch( const std::string &str, const std::string &qry )
 {
-    if( std::locale().name() != "en_US.UTF-8" && std::locale().name() != "C" ) {
-        auto &f = std::use_facet<std::ctype<wchar_t>>( std::locale() );
+    std::locale temp_locale = std::locale();
```

Comment:
```suggestion
    auto temp_locale = std::locale{};
```
we should probably use `auto` when it's obvious.